src/sentry.ts

```diff
@@ -27,9 +25,11 @@ const sentryClient = Sentry.init({
     release,
     environment,
     skipOpenTelemetrySetup: true,
-    tracesSampleRate: 1.0,
```

Comment:
Removing this leads to Sentry not adding non-essential OTEL instrumentation out of the box.